chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -226,6 +226,13 @@
   action :create
 end
 
+file "#{logdir}/production.log" do
+  owner 'crowbar'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.